app/models/extract_data.py

```
@@ -12,4 +13,10 @@ class ExtractDataModel(BaseModel):
     user_id: int = Field(..., alias="userId")
     url: str
 
-    model_config = ConfigDict(populate_by_name=True) 
\ No newline at end of file
+    model_config = ConfigDict(populate_by_name=True)
+
+    # SQLAlchemy Index
+    __table_args__ = (
+        Index("idx_extract_data_status", "status"),
+        Index("idx_extract_data_user_status", "user_id", "status"),
+    ) 
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)